src/fallout.coffee

```diff
@@ -63,15 +60,12 @@
     onFalloutStoreChange: ->
       for name, selector of @_falloutWatchSelectors()
         storeValue = Fallout.store.data.getIn selector
-        unless @state["_#{name}"] is storeValue
+        unless Immutable.is @state["_#{name}"], storeValue
```

Comment:
Are you comparing Immutable values here? If so you might want Immutable.is instead of ===